src/css/profile/mobile/common/toast.less

```diff
@@ -2,20 +2,19 @@
 /* Popup Toast */
 .ui-popup.ui-popup-toast {
 	background-color: transparent;
+	margin-bottom: 0;
```

Comment:
Is the margin-bottom value 0 correct? Make sure that the spec comes out at the bottom and fits.

Reply:
Margin bottom described in the spec is set in line 12.  We don't need additional margin.

Reply:
@korneliakobiela 
Are you setting margins elsewhere?
If not, it is better to delete margin-bottom.

Reply:
Are you setting margins elsewhere?
If not, it is better to delete margin-bottom.

Reply:
We need it to remove margins from `ui-popup` class.